app/search_builders/sufia/find_works_search_builder.rb

```diff
@@ -0,0 +1,44 @@
+class Sufia::FindWorksSearchBuilder < Sufia::SearchBuilder
+  include Sufia::MySearchBuilderBehavior
```

Comment:
Add a comment here about why you're including this.  Something like `# Only search for works that we have edit access to`